### DISCUSSION TITLE
PopUP ou mesage alerte sur ressource

### DISCUSSION THREAD

**Discussion by taib27** [JeromeDevome/GRR#368] [General]
Bonjour , 
Je cherche à mettre en place une sorte de popup et/ou message qui s'affiche à la sélection d'une ressource pour signaler les prérequis à la réservation .
est-ce qu'il existe à moyen pour faire cela ?!  Sinon : une idée pour mettre cela en place .

Merci par avance.

**Comment by ynaessens** [comment]
Bonjour,
quelle version de GRR utilisez-vous ?
Dans quelle page voudriez-vous mettre en place ce pop-up ?
Cordialement,
YN

**Reply by taib27** [reply]
Re :²
GRR : Version 4.2.2
J'ai un domaine SALLE avec 4 ressources " Salle1   Salle2    Salle3    Salle4. "
La salle1 est particulière les utilisateurs doivent récupérer des outils dans mon bureau pour pouvoir l'utiliser.

Du coup, je cherchais un moyen " Pop-up ou message d'alerte" qui s'affiche une fois la source est sélectionner. 
Entre autres avant l’accès au menu réservation , une sorte de boîte avec bouton OK pour accéder au champs réservation de la source en question .

Merci pour tout

**Reply by ynaessens** [reply]
Une solution, mais pas sous forme de pop-up, est la description complète de la ressource, que l'on peut faire afficher dans l'entête du planning.
Pour inclure un pop-up dans la page d'édition des réservations, ce n'est pas facile dans la mesure où il est possible de réserver plusieurs ressources simultanément. À moins qu'il ne soit possible d'afficher un pop-up au survol du nom de la ressource dans le sélecteur ?

**Reply by taib27** [reply]
Le champs **description complète** :   je l'utilise pour **la justification** de la réservation car elle peut être soumise à validation " J'avoue que c'est une solution de contournement".
afficher **un pop-up au survol du nom de la ressource dans le sélecteur** ? : Si c'est possible je suis preneur . 💯 

Le popup ne servira que pour de l'information rien d'autre "aucun effet sur l'accès à la ressource ".

Merci par avance

**Reply by ynaessens** [reply]
Bonjour,
je pensais à la description complète de la ressource, pas celle de la réservation.
Pour ce qui est du popup dans le sélecteur, je n'ai pas de solution toute faite, ce qui n'empêche pas de chercher ;-)
Cordialement 
YN

**Reply by taib27** [reply]
si si autant pour moi ... c'était bien celle de la ressource .

**Reply by ynaessens** [reply]
Alors je ne comprends plus ce que vous faites : l'utilisateur ne peut pas modifier le description de la ressource, alors que la description complète de la réservation est un champ modifiable.
Donc, dans l'état actuel des choses, le champ description complète de la ressource peut servir à afficher un message avec une consigne particulière et le champ description complète de la réservation peut servir à justifier la demande de réservation (qui, j'imagine, sera modérée par un gestionnaire/administrateur).
Qu'en dites-vous ?

**Reply by taib27** [reply]
une image est meilleur que mille mot ^_^
![Capture](https://github.com/JeromeDevome/GRR/assets/71597810/c3841040-0612-4824-93b0-2563eccd3bfa)

 la description complète de la réservation est un champ modifiable : YES le user doit renseigner let justifier la résa .
 
 désolé si je me suis mal exprimer . 
 Merci encor une fois pour votre aide et investissement .
